docs/pages/versions/unversioned/sdk/screen-capture.md

```diff
@@ -46,57 +46,59 @@ export default function ScreenCaptureExample {
 ### Example: functions
 
 ```javascript
-import { preventScreenCaptureAsync, allowScreenCaptureAsync } from 'expo-screen-capture';
+import * as ScreenCapture from 'expo-screen-capture';
 import React from 'react';
 import { Button, View } from 'react-native';
 
 export default class ScreenCaptureExample extends React.Component {
+  componentDidMount() {
+    /* @info The provided function will be run whenever a screenshot of your app is taken. */
+    ScreenCapture.addScreenshotListener(() => {
+      alert('Thanks for screenshotting my beautiful app 😊');
+    }); /* @end */
+  }
+
   render() {
     return (
       <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
         <Button onPress={this._activate}>Activate</Button>
         <Button onPress={this._deactivate}>Deactivate</Button>
+        <Button onPress={this._listen}>Listen for screenshots</Button>
```

Comment:
Hm looks like this` _listen` function isn't defined

Reply:
removed 👍